src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -439,7 +462,8 @@ async def _exec_batch(
         while completed_line < total_lines:
             done, pending = await asyncio.wait(pending, return_when=asyncio.FIRST_COMPLETED)
             completed_line_results = [task.result() for task in done]
-            self._persist_run_info(completed_line_results)
+            if not isinstance(self._executor_proxy, ChatGroupOrchestratorProxy):
```

Comment:
Why we have this validation here?

Reply:
line_result structure changed for chat group scenario. Need to handle related process line result function later.